File: tests/fixtures/expected/sample.c
#include <stdio.h>

 


int main(void) {
    
    int x = 42;  
    char *msg = "Hello, // not a comment";
    char *msg2 = "Hello, /* also not */ a comment";
    char c = '\''; 
    char d = '/';  

    


    printf("%s\n", msg);

    return 0; 
}

File: tests/fixtures/expected/sample.py
#!/usr/bin/env python3

import sys  

def hello():
    """This is a docstring - NOT a comment"""
    x = "hello # not a comment"
    y = 'world # also not a comment'
    z = '''triple
    single # not a comment
    '''
    
    print(x + y)  


hello()
